NEWS

```diff
@@ -19,7 +19,7 @@ PHP                                                                        NEWS
   . Fixed zend call stack size for macOs/arm64. (David Carlier)
   . Added support for Zend Max Execution Timers on FreeBSD. (Kévin Dunglas)
   . Ensure fiber stack is not backed by THP. (crrodriguez)
-  . Implement GH-13609 (Dump wrapped object in WeakReference class). (nielsdos)
+  . Implemented GH-13609 (Dump wrapped object in WeakReference class). (nielsdos)
```

Comment:
This file has an 80 character width limit, can you reformat the changed lines to reflect that?

Reply:
Done